### PR TITLE
Templates for various issue

### PR DESCRIPTION
**Pull request by tomas-kulhanek**

### CODE REVIEW

**Comment by tomas-kulhanek** [comment]
@ollie jsi taky pro toto?

File: .github/ISSUE_TEMPLATE/1_Bug_report.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        Please, take the time to show us some code and/or config that is needed for others to reproduce the problem easily.
+    validations:
+      required: true
+  - type: textarea
+    id: possible-solution
+    attributes:
+      label: Possible Solution
+      description: |
+        Optional: only if you have suggestions on a fix/reason for the bug
+        Don't hesitate to create a pull request with your solution, it helps get faster feedback.
```
Comment (ollie) [inline .github/ISSUE_TEMPLATE/1_Bug_report.yaml:36]: I'd add something like this "… feedback. Please make pull requests as small as possible, don't mix various changes (e.g. adding a feature _and_ fixing code style in a lot of places), as it makes review much harder and we will most likely want you to separate the changes."

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (ollie) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: We have changelog not in the src dir, don't even have the upgrade log, so maybe that might need tweaking.

**Review by ollie — COMMENTED** [review]
Overall I think this is a bit of an overkill at the moment, but I also don't see any reason why not to do it, so I kinda agree on it. But @edariedl  has final say.

**Review by edariedl — COMMENTED** [review]
We had about 10 issues and 20 open PRs over the 8 years this library lifespan.

This is quite an overkill and we don't need it right know. Which doesn't mean we won't add something like this in the future if we find it necessary.

**Comment by tomas-kulhanek** [comment]
@edariedl  @ollie  Okay. But I think a template for the bug and an external link to the documentation would be great. So that people immediately write, for example, how the problem can be reproduced, etc. This will reduce the time needed to understand the problem.

**Pull request closed without merging**